### DISCUSSION TITLE
A few questions about the upgrade guide

### DISCUSSION THREAD

**Discussion by alexandergitter** [shakacode/shakapacker#172] [General]
Hi there,

I'm currently going through the upgrade guide (coming from Webpacker v5) and I have a few questions.

- The upgrade guide talks about going from v5 to v6.0.0.rc.6 in multiple places, but the code snippet then actually uses `6.0.0.rc.13`. Which is the correct version to use? There's also a note saying "[Check the gem page to verify the latest version](https://rubygems.org/gems/shakapacker)", which is 6.5.0. I'm assuming this refers to the latest RC version? (Which would be `6.0.0.rc.14`)

- Step 9 states that `config/webpack/environment.js` was changed to `config/webpack/base.js` - however this file does not get created when running `bundle exec rails webpacker:install`. Is that on purpose? What about the environment specific files (`development.js`, `production.js`, `test.js`), do we need to keep these?

- The snippet in Step 9 contains the following line: `const { webpackConfig, merge } = require('@rails/webpacker');`. The dependency in package.json is called `shakapacker`, so I'm assuming it should be `const { webpackConfig, merge } = require('shakapacker');` instead?

Thanks!

**Comment by justin808** [comment]
I moved this to #191. 

PR's welcome!

**Comment by ahangarha** [comment]
Corrected by https://github.com/shakacode/shakapacker/pull/222
